src/components/RoomNameInput/roomNameInputPropTypes.js

```diff
@@ -26,6 +39,11 @@ const defaultProps = {
     disabled: false,
     errorText: '',
     forwardedRef: () => {},
+
+    inputID: undefined,
+    onBlur: () => {},
+    defaultValue: CONST.POLICY.ROOM_PREFIX,
```

Comment:
@fedirjh Any reason we give the default value `#`?

Reply:
```jsx
value={this.props.value.substring(1)} // Since the room name always starts with a prefix, we omit the first character to avoid displaying it twice.
```
@mollfpr as stated here every room name should start with `#` , so I taught default value should be `#`

Reply:
@fedirjh Yes, the room name value will always have `#` but I don't think the default value for `defaultValue` should be set to the prefix. The prefix `#` is handled in `setModifiedRoomName` so I think we don't need to set it in the props.

What do you think @puneetlath?

Reply:
If it doesn't have any practical effect, then I think we can remove.

Reply:
@fedirjh Let's set it to `undefined`.

Reply:
done!